delivery/src/on_package_created/main.py

```diff
@@ -80,6 +81,7 @@ def save_shipping_request(order: dict) -> None:
             "message": "Cannot update shipping request in status '{}'".format(result["Item"]["status"]),
             "orderId": order["orderId"]
         })
+        log_metrics("ecommerce.deliveries", "deliveryCreateFailed", 1)
```

Comment:
This branch is reached mostly when a duplicate event is received (this is why the loglevel is __info__, not __warn__). What is the value of having a metric for these types of situation?

Reply:
Fair enough, for the current implementation the deliveryCreateFailed metric would not be that useful.  I was thinking that in a real world e-commerce system, not begin able to initiate a delivery job would be a valuable indicator.